### PR TITLE
Add notes about quote macro as explained in #587

### PR DESCRIPTION
**Pull request by Koziolek**

### CODE REVIEW

**Comment by Koziolek** [comment]
@scouten @doomspork could you check this?

**Comment by scouten** [comment]
Looks good to me.

File: lessons/basics/modules.md [hunk 1]
```diff
@@ -252,3 +252,5 @@ end
 The `Ecto.Migration.__using__/1` macro includes an import call so that when you `use Ecto.Migration` you also `import Ecto.Migration`. It also sets up a module property which we will assume controls Ecto’s behavior.
 
 To recap: the use macro simply invokes the `__using__/1` macro of the specified module. To really understand what that does you need to read the `__using__/1` macro.
+
+**Note**: `quote` is macro used when we work with [metaprogramming](../advanced/metaprogramming).
```
Comment (doomspork) [inline lessons/basics/modules.md]: I think we're missing something here, what about:

```markdown
**Note**: `quote` is a macro used when we work with [metaprogramming](../advanced/metaprogramming)
```

**Comment by doomspork** [comment]
@Koziolek if you could update this we're good to merge :)

**Comment by Koziolek** [comment]
I'll update this today at night CET

**Comment by Koziolek** [comment]
Done.

**Comment by doomspork** [comment]
Thank you @Koziolek!

**Pull request merged**